### ISSUE TITLE
Fastlane Update Broke Notarization

### ISSUE THREAD

**Issue by mikeKane** [fastlane/fastlane#16710]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Updating to Fastlane 2.149.1 broke Mac notarization.

##### Command executed

fastlane mac beta

```
INFO [2020-06-30 12:36:51.78]: Uploading package to notarization service, might take a while
WARN [2020-06-30 12:36:53.16]: Lane Context:
INFO [2020-06-30 12:36:53.16]: {:PLATFORM_NAME=>:mac, :LANE_NAME=>"mac beta", :VERSION_NUMBER=>"10.0.0"}
ERROR [2020-06-30 12:36:53.16]: Shell command exited with exit status 1 instead of 0.
INFO [2020-06-30 12:36:53.16]: Successfully generated documentation at path './README.md'

+------+--------------------+-------------+
|            fastlane summary             |
+------+--------------------+-------------+
| Step | Action             | Time (in s) |
+------+--------------------+-------------+
| 1    | get_version_number | 0           |
| 💥   | notarize           | 32          |
+------+--------------------+-------------+

ERROR [2020-06-30 12:36:53.18]: fastlane finished with errors

/Users/USR/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': [!] Shell command exited with exit status 1 instead of 0. (FastlaneCore::Interface::FastlaneShellError)

```

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>```INFO [2020-06-30 12:44:11.22]: Uploading package to notarization service, might take a while
WARN [2020-06-30 12:44:12.40]: Lane Context:
INFO [2020-06-30 12:44:12.40]: {:PLATFORM_NAME=>:mac, :LANE_NAME=>"mac beta", :VERSION_NUMBER=>"10.1.0"}
ERROR [2020-06-30 12:44:12.40]: Shell command exited with exit status 1 instead of 0.
INFO [2020-06-30 12:44:12.40]: Successfully generated documentation at path '/Users/connectifyjenkins/src/webrtc-ios/src/third_party/connectify/mac/Speedify/fastlane/README.md'

+------+--------------------+-------------+
|            fastlane summary             |
+------+--------------------+-------------+
| Step | Action             | Time (in s) |
+------+--------------------+-------------+
| 1    | get_version_number | 0           |
| 💥   | notarize           | 8           |
+------+--------------------+-------------+

ERROR [2020-06-30 12:44:12.41]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...
Search query: Shell command exited with exit status 1 instead of 0.

URL: https://api.github.com/search/issues?q=Shell%20command%20exited%20with%20exit%20status%201%20instead%20of%200.+repo:fastlane/fastlane
➡️  Fastlane Update Broke Notarization
    https://github.com/fastlane/fastlane/issues/16710 [open] 0 💬
    53 seconds ago

➡️  Improve Beta Setup
    https://github.com/fastlane/fastlane/pull/1 [closed] 1 💬
    04 Feb 2017

➡️  upload_symbols_to_crashlytics not working with Firebase/Crashlytics
    https://github.com/fastlane/fastlane/issues/16168 [closed] 21 💬
    a day ago

and 78 more at: https://github.com/fastlane/fastlane/search?q=Shell%20command%20exited%20with%20exit%20status%201%20instead%20of%200.&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
Traceback (most recent call last):
        32: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/ruby_executable_hooks:24:in `<main>'
        31: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/ruby_executable_hooks:24:in `eval'
        30: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/fastlane:23:in `<main>'
        29: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/fastlane:23:in `load'
        28: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/bin/fastlane:23:in `<top (required)>'
        27: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
        26: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
        25: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
        24: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
        23: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
        22: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
        21: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
        20: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
        19: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
        18: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
        17: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
        16: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
        15: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
        14: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
        13: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/lane.rb:33:in `call'
        12: from Fastfile:27:in `block (2 levels) in parsing_binding'
        11: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
        10: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
         9: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
         8: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:227:in `chdir'
         7: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:253:in `block in execute_action'
         6: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
         5: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:261:in `block (2 levels) in execute_action'
         4: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/actions/notarize.rb:42:in `run'
         3: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/helper/sh_helper.rb:12:in `sh'
         2: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/helper/sh_helper.rb:80:in `sh_control_output'
         1: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
/Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': Shell command exited with exit status 1 instead of 0. (FastlaneCore::Interface::FastlaneShellError)
        32: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/ruby_executable_hooks:24:in `<main>'
        31: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/ruby_executable_hooks:24:in `eval'
        30: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/fastlane:23:in `<main>'
        29: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/bin/fastlane:23:in `load'
        28: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/bin/fastlane:23:in `<top (required)>'
        27: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
        26: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
        25: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
        24: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
        23: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
        22: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
        21: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
        20: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
        19: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
        18: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
        17: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
        16: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
        15: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
        14: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
        13: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/lane.rb:33:in `call'
        12: from Fastfile:27:in `block (2 levels) in parsing_binding'
        11: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
        10: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
         9: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:227:in `execute_action'
         8: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:227:in `chdir'
         7: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:253:in `block in execute_action'
         6: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
         5: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/runner.rb:261:in `block (2 levels) in execute_action'
         4: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/actions/notarize.rb:42:in `run'
         3: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/helper/sh_helper.rb:12:in `sh'
         2: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane/lib/fastlane/helper/sh_helper.rb:80:in `sh_control_output'
         1: from /Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
/Users/connectifyjenkins/.rvm/gems/ruby-2.6.0/gems/fastlane-2.149.1/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': [!] Shell command exited with exit status 1 instead of 0. (FastlaneCore::Interface::FastlaneShellError)``` </pre>
</details>

### Environment

<details>
  <pre> <details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.3                                     |
| Ruby                        | 2.6.0                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.24.3 (Apple Git-128)          |
| Installation Source         | ~/.rvm/gems/ruby-2.6.0/bin/fastlane         |
| Host                        | Mac OS X 10.15.3 (19D76)                    |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.6.0/lib                |
| OpenSSL Version             | OpenSSL 1.1.1g  21 Apr 2020                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.5                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

#Issues: https://github.com/fastlane/fastlane/issues/15184


platform :mac do 
  lane :beta do |options|
    ENV["FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD"] = options[:secret_password]    #build_app(workspace: "MYAPP.xcworkspace", scheme: "MYSCHEME")
    #upload_to_testflight
    version = get_version_number(
  	xcodeproj: "./MYAPP.xcodeproj",
 	 target: "MYSCHEME"
    ) 
    notarize(
        package:options[:dmg_path],
        bundle_id: "MYBUNDLEID"    
    )
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("MYBUNNDLEID") # The bundle identifier of your app
apple_id("MYEMAIL@gmail.com") # Your Apple email address

itc_team_id("TEAM") # App Store Connect Team ID
team_id("TEAMID") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.149.1 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| executable-hooks          | 1.6.0        |
| bundler-unload            | 1.0.2        |
| rubygems-bundler          | 1.4.5        |
| bundler                   | 2.1.4        |
| CFPropertyList            | 3.0.2        |
| public_suffix             | 2.0.5        |
| addressable               | 2.7.0        |
| atomos                    | 0.1.3        |
| aws-eventstream           | 1.1.0        |
| aws-partitions            | 1.337.0      |
| aws-sigv4                 | 1.2.1        |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.102.1      |
| aws-sdk-kms               | 1.35.0       |
| aws-sdk-s3                | 1.72.0       |
| babosa                    | 1.0.3        |
| claide                    | 1.0.3        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| digest-crc                | 0.5.1        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| dotenv                    | 2.7.5        |
| emoji_regex               | 1.0.1        |
| excon                     | 0.75.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 1.0.1        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.2       |
| multi_json                | 1.14.1       |
| os                        | 1.1.0        |
| signet                    | 0.14.0       |
| googleauth                | 0.13.0       |
| httpclient                | 2.8.3        |
| mini_mime                 | 1.0.2        |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.2        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.26.2       |
| json                      | 2.3.1        |
| mini_magick               | 4.10.1       |
| multi_xml                 | 0.6.0        |
| plist                     | 3.5.0        |
| rubyzip                   | 1.3.0        |
| security                  | 0.1.3        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| slack-notifier            | 2.3.2        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.8.0        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.17.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| xcpretty-travis-formatter | 1.0.0        |
</details>

*generated on:* **2020-06-30**
</details> </pre>
</details>

If I notarize manually it works just fine

**Comment by joshdholtz** [comment]
@mikeKane Hey! Can you re-run with verbose on for notarize? 👇 
```rb
notarize(verbose: true)
```

Its hard to see what failed here 🤔 What version you were on before `2.149.1`?

**Comment by odrobnik** [comment]
Could this be the same problem as #16770 ?

**Comment by robbmcmahan** [comment]
I'm having the same issue on Jenkins using `Xcode 11.5` and fastlane `2.149.1`. Also a successful response from Apple. This is what i'm getting:

```
shell_error Exit status of command xcrun stapler staple "xxx/path/pkg"' was 68 instead of 0

error is Error Domain=kCFErrorDomainCFNetwork Code=310 "(null)" UserInfo={_kCFStreamErrorCodeKey=-2096, _kCFStreamErrorDomainKey=4}
CloudKit's response is inconsistent with expections: (null)
The staple and validate action failed! Error 68
```

**Comment by anyuser** [comment]
I had the same issue with notarization. But it was not caused by a fastlane update, but by an updated agreement on app store connect. With `verbose: true` it showed this error message:
```
You must first sign the relevant contracts online. (1048)
```
After accepting the new agreements in app store connect it worked again. Maybe it would be good to show this error without verbose logging.

**Comment by joshdholtz** [comment]
This should be fixed in `2.151.1` 💪
